src/controllers/campus.ctrl.ts

```diff
@@ -38,41 +39,66 @@ module CampusController {
    * @param  {restify.Response} res
    * @param  {restify.Next}     next
    */
+
   export function createCampus(req: restify.Request, res: restify.Response, next: restify.Next) {
-    var campusInfo = req.body;
-    var requestValid = pv.verifyParams(
-                        campusInfo.name,
-                        campusInfo.address,
-                        campusInfo.address.address,
-                        campusInfo.address.geoCode,
-                        campusInfo.address.geoCode.lat,
-                        campusInfo.address.geoCode.long);
-
-    if (!requestValid) {
-      res.send(400, {"message": "Bad Request: invalid or missing paramters"});
-      next();
-      return;
-    }
+    auth.checkAdmin(req).then( (isAdmin) => {
```

Comment:
Can you drop this then down to the next line.  Makes it a little easier to read.